release-calendar/src/client/api-service.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
+import {Channel, Release as ReleaseEntity} from '../types';
```

Comment:
Also add `ApiService` interface to list of imports

Reply:
Ohh, haha, i was wondering about that. Added!

Reply:
But also changed to ApiServiceInterface. Otherwise it isn't getting recognized.